### PR TITLE
Adding user guide: mining with docker and xmrig

### PR DESCRIPTION
**Pull request by b-i-t-n**
Adding a guide for mining with xmrig within an Alpine Linux docker container.

If this is not something we are looking for in a guide let me know but as it allows a miner to work cross distribution without worrying about dependencies or how to compile it. 

I thought it would be good for new linux users.

### CODE REVIEW

File: resources/user-guides/mining_with_xmrig_and_docker.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+##Introduction
+
+This guide is two fold, ease of use for mining on Linux distributions and some extra security around mining as most of these miners have not had security auditing.
+
+At the end of this guide you will be able to sleep a little easier knowing that if the miner gets exploited it will not migrate to your OS.
+
+###Why Docker
+
+Docker is being used as it is the most well known and has the biggest chance to be already installed.
```
Comment (mattcode55) [inline resources/user-guides/mining_with_xmrig_and_docker.md]: Maybe you could link to the Docker documentation for people that don't already have it installed.

File: resources/user-guides/mining_with_xmrig_and_docker.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+title: "Mining with XMRig and Docker"
+title-pre-kick: "Mining XMR"
+title-kick: "With XMRig and DOcker"
+title-post-kick: ""
+kick-class: "purple-kicks"
+icon: "icon_userguides"
+attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and is licensed under Creative Commons BY 3.0 -->"
+---
+
+##Introduction
```
Comment (MaxXor) [inline resources/user-guides/mining_with_xmrig_and_docker.md]: Please add spaces between `##` and the titles to have valid formatted titles. Same applies to the other ones below.

**Comment by b-i-t-n** [comment]
Sorry if I am slow to update, free time is hard to come by.
How is that?

**Comment by mattcode55** [comment]
Also you might want to add a link to this guide in `/resources/user-guides/index.md`

File: resources/user-guides/index.md [hunk 3]
```diff
@@ -69,6 +69,8 @@ title: "User Guides"
 
 User Guides for Mining coming soon.
```
Comment (MaxXor) [inline resources/user-guides/index.md]: You can remove this text now.

**Comment by SamsungGalaxyPlayer** [comment]
@b-i-t-n can you reword the guide to exclude first person?

**Comment by fluffypony** [comment]
@SamsungGalaxyPlayer merged this to prevent it waiting forever, someone will have to fix it in a subsequent PR

**Pull request merged**